summertime/model/single_doc/base_single_doc_model.py

```diff
@@ -34,3 +34,20 @@ def assert_summ_input_type(cls, corpus, query):
                 raise TypeError(
                     "Query-based single-document summarization requires query of `List[str]`."
                 )
+
+    @classmethod
+    def assert_summ_input_language(cls, corpus, query):
+
+        warning = "Warning: non-ASCII input corpus detected!\n\
+                    If this is not English, consider using \
+                    one of our multilingual models."
```

Comment:
Maybe add `such as [summertime.model.multilingual.xxx]`?

Reply:
Addressed this in new commits!